extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1046,6 +1054,7 @@ export class AmpA4A extends AMP.BaseElement {
    * @private
    */
   renderNonAmpCreative_() {
+    this.promiseErrorHandler_(new Error('fallback to 3p'));
```

Comment:
I'm reporting these as "expected". The goal, for now, to get the baseline and monitor how it changes between deployments. It's totally reasonable later to remove it.